### ISSUE TITLE
Server start failed (can't find angular.json)

### ISSUE THREAD

**Issue by cgs-lbergeron** [lucono/karma-test-explorer#42]
Since version 0.7.2, I get this error when karma starts :

```
------------------------------------
Karma Test Explorer: Launching Karma
------------------------------------

29 07 2022 11:37:39.750:ERROR [karma-server]: Server start failed on port 9976: Error: The '@angular-devkit/build-angular/plugins/karma' karma plugin is meant to be used from within Angular CLI and will not work correctly outside of it.
```

I reverted to 0.7.1 and I don't get this error

I've set the log level of the extention to trace and it seems it can't find my angular.json : 

`[2022-07-29 15:52:22.253] [DEBUG] [ProjectFactory]: Cannot get Angular projects - Angular Json file does not exist: /Program Files/Microsoft VS Code/C:/scan/git/scan/Scan Factory Intelligence GUI NG App/angular.json`

**Comment by lucono** [comment]
This is most likely a duplicate of #43. Please also provide:
- Your environment details
- You project folder structure
- Your Karma Test Explorer extension settings

**Comment by lucono** [comment]
I've released an update (`v0.7.3`) to the marketplace. Update to the new version in VSCode and confirm if the issue is addressed.

**Comment by cgs-lbergeron** [comment]
@lucono Thank you, `v0.7.3` fixes it !
